### ISSUE TITLE
Edge Login Issue

### ISSUE THREAD

**Issue by pvagnini** [asadsahi/AspNetCoreSpa#192]
Hi Asadsahi,

Deployment works fine with Chrome, but when using Edge when clicking Login it gives the error;

Error Getting Token
Reference Error:
'URLSearchParams' is not defined.

I hope you can help.

Thanks
Paolo

**Comment by asadsahi** [comment]
@pvagnini can you try in current branch as I have removed this logic to simplify data service.

**Comment by pvagnini** [comment]
Hi Asadsahi,

The current Master branch does not appear to work. I note that files like package.json appear to be missing?

Regards
Paolo

**Comment by asadsahi** [comment]
@pvagnini  thanks for pointing this out. Sorry for confusion. Yesterday I have merged changes that include complete angular cli branch. I have update the [installation instructions ](https://github.com/asadsahi/AspNetCoreSpa#installation)and a [note ](https://github.com/asadsahi/AspNetCoreSpa#note-this-is-the-new-angular-cli-based-branch-where-clientapp-is-completely-managed-by-angular-cli-if-you-are-interested-in-previous-cli-branch-where-integration-of-angular-cli-was-minimal-only-for-code-generation-then-switch-to-angular-non-cli-branch)to clarify two different branches. Hope this clarifies.

In short you need to cd into ClientApp to do npm install now.

**Comment by pvagnini** [comment]
Thanks Asadsahi,

The login the Edge now works.

Much quicker startup in Visual Studio now.

Great work thanks.
